src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/NomisService.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.justice.digital.hmpps.whereabouts.services;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+
+@Service
+public class NomisService {
+
+    private RestTemplate restTemplate;
+
+    public NomisService(@Qualifier("elite2ApiRestTemplate") RestTemplate restTemplate) {
+        this.restTemplate = restTemplate;
+    }
+
+    public void updateAttendance(String offenderNo, long activityId, String eventOutcome, String performance) {
+        final var url = "/bookings/offenderNo/{offenderNo}/activities/{activityId}/attendance";
+        if (performance != null)
+            restTemplate.put(url, Map.of("eventOutcome", eventOutcome, "performance", performance), offenderNo, activityId);
```

Comment:
`Map.of("eventOutcome", eventOutcome, "performance", performance)` should be a variable and then only need one call to `restTemplate.put` in the code